lib/quadrigacx/client/private.rb

```diff
@@ -56,6 +56,13 @@ def open_orders params={}
       request(:post, '/open_orders', params)
     end
 
+    # Returns JSON list of details about 1 or more orders.
+    #
+    # id – a single or array of 64 characters long hexadecimal string taken from the list of orders
```

Comment:
Super minor nit-pick, but can we add a `.` at the end of this line? Just so we have consistency with all the other arguments documentation.